src/components/RuleCard/index.jsx

```diff
@@ -185,25 +201,19 @@ function RuleCard({
           classes={{ avatar: classes.cardHeaderAvatar }}
           className={classes.cardHeader}
           avatar={
-            Number.isInteger(Number(rule.priority)) && (
+            Number.isInteger(Number(headerPriority)) && (
               <Fragment>
                 <Avatar
-                  title="Priority"
-                  aria-label="Priority"
-                  className={classes.avatar}>
+                  title={priorityTitle}
+                  aria-label={priorityTitle}
+                  className={classNames(
+                    classes.avatar,
+                    classes.scheduledPriorityChange
```

Comment:
I think we only want to add `scheduledPriorityChange` if we're showing a scheduled change priority. Something like:

```jsx
classNames(
  classes.avatar,
  { [classes.scheduledPriorityChange]: isScheduledChange }
)
```

Reply:
Whoops, sorry about that!